cel/program.go

```diff
@@ -26,9 +26,16 @@ import (
 type Program interface {
 	// Eval returns the result of an evaluation of the Ast and environment against the input vars.
 	//
-	// If the evaluation is an error, the result will be nil with a non-nil error.
+	// If the OptTrackState or OptExhaustiveEval is used, the `details` response will be non-nil.
```

Comment:
Nit: "If the _`x`_ or _`y`_ **flags(?) are** used..." (notice the back quote mark on x/y)